### PR TITLE
Bar

### PR DESCRIPTION
**Pull request by sjurgemeyer**

### CODE REVIEW

**Comment by acm1** [comment]
too many commits -- rebase

File: test.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Top line
```
Comment (lbredeso) [inline test.txt]: I prefer the term "First line".

File: test.txt [hunk 2]
```diff
@@ -5,10 +6,13 @@ The weapons must have disrupted our communicators.
 You saw something as tasty as meat, but inorganically materialized out of patterns used by our transporters.
 Captain, the most elementary and valuable statement in science, the beginning of wisdom, is 'I do not know.' All transporters off.
 Now what are the possibilities of warp drive? Cmdr Riker's nervous system has been invaded by an unknown microorganism.
+new line
```
Comment (sjurgemeyer) [inline test.txt]: @acm1 what do you think?

Comment (acm1) [inline test.txt]: seems accurate

Comment (robzienert) [inline test.txt]: nope

**Pull request closed without merging**